app/views/schools/users/show.html.erb

```diff
@@ -0,0 +1,183 @@
+            <% end %>
+          </div>
+        <% end %>
+        <% if @presenter.courses_coached.present? %>
+          <h2 class="mt-6 text-xl font-bold"><%= t("schools.users.show.coaches_in") %></h2>
+          <div class="mt-4 flex flex-col gap-4">
+            <% @presenter.courses_coached.each do |course| %>
+              <div class="flex items-center justify-between bg-white border border-gray-200 rounded-lg p-4">
+                <div class="flex items-center gap-3 overflow-hidden" data-test-class="users-courses_coached">
+                  <div class="w-20 border border-gray-200 rounded-xl">
+                    <div class="w-full pt-1/2 relative">
+                      <% if course.thumbnail_url.present? %>
+                        <img src="<%= course.thumbnail_url  %>" alt="Course image" class="absolute top-0 object-cover rounded-lg">
+                      <% else %>
+                        <div class="svg-bg-pattern-2 absolute inset-0 rounded-xl object-cover svg-bg-pattern-1"></div>
+                      <% end %>
+                    </div>
+                  </div>
+                  <p class="text-base font-semibold"><%= course.name %></p>
+                </div>
+                <%= link_to school_course_coaches_path(course.id), class: "hover:text-primary-500 hover:text-lg" do %>
+                  <i class="if i-external-link-light if-fw" ></i>
+                <% end %>
+              </div>
+            <% end %>
+          </div>
+        <% end %>
+        <% if @presenter.courses_authored.present? %>
+          <h2 class="mt-6 text-xl font-bold"><%= t("schools.users.show.authors_in") %></h2>
+          <div class="mt-4 flex flex-col gap-4">
+            <% @presenter.courses_authored.each do |course| %>
+              <div class="flex items-center justify-between bg-white border border-gray-200 rounded-lg p-4">
+                <div class="flex items-center gap-3 overflow-hidden" data-test-class="users-courses_authored">
+                  <div class="w-20 border border-gray-200 rounded-xl">
+                    <div class="w-full pt-1/2 relative">
+                      <% if course.thumbnail_url.present? %>
+                        <img src="<%= course.thumbnail_url  %>" alt="Course image" class="absolute top-0 object-cover rounded-lg">
+                      <% else %>
+                        <div class="svg-bg-pattern-2 absolute inset-0 rounded-xl object-cover svg-bg-pattern-1"></div>
+                      <% end %>
+                    </div>
+                  </div>
+                  <p class="text-base font-semibold"><%= course.name %></p>
+                </div>
+                <%= link_to authors_school_course_path(course.id), class: "hover:text-primary-500 hover:text-lg" do %>
+                  <i class="if i-external-link-light if-fw" ></i>
+                <% end %>
+              </div>
+            <% end %>
+          </div>
+        <% end %>
+      </div>
+    </div>
+  </div>
+</div>
+</div>
```

Comment:
These two closing div-s shouldn't be at the same level. You may have a structure mistake, or probably just an extra closing div.